guides/doc-Upgrading_Project/topics/proc_upgrading-a-disconnected-project-server.adoc

```diff
@@ -93,7 +93,7 @@ Do not install or update any packages at this stage.
 .. Edit the `/etc/yum.repos.d/satellite.repo` file:
 +
 ----
-# vi /etc/yum.repos.d/sat6.repo
+# vi /etc/yum.repos.d/satellite.repo
```

Comment:
You could also use `{project-context}` if you add the "allow attributes in the code block" line.

Reply:
Disconnected ones are downstream exclusive, no?

Reply:
I didn't use attributes for a single build exclusive file. (But I can if required).